common/src/main/java/com/wynntils/wc/utils/WynnItemMatchers.java

```diff
@@ -94,19 +107,13 @@ public static boolean isCosmetic(ItemStack stack) {
     }
 
     public static boolean isMythic(ItemStack stack) {
+        if (!isUnidentified(stack) && !isGear(stack)) return false;
```

Comment:
This logic is making my head spin. Can you de Morgan it, and possibly move the code below into an if statement? 

This code is not missing a too small amount of non-existing inverted logic negations. Or the other way round.